### ISSUE TITLE
一个未来大版本的畅想!2.8!天马行空的创新.

### ISSUE THREAD

**Issue by superDMS** [yingDev/WGestures2-mac-bugs#96]
就是基于现有的手势，自动自适应能生成一张脑图一样的管理界面,然后像管理脑图一样增删手势,fastgesture的手势分支提示图有点这个味道 ![image](https://user-images.githubusercontent.com/57837939/144698150-e1d05e62-e3b5-4f51-96b8-d835f7515da6.png)
或者是自动自适应一个全局的手势导览(手势越多ui越密集,越少越ui越大,就像win+tab的那个任务管理界面的逻辑,元素越多按钮越小,自动排列),且有一个全局呼出脑图的热键让脑图透明浮空在界面之上，然后用光标直接选择想要触发的手势。就好比有些游戏中按住游戏手柄(或键盘)上的某个按键就能呼出一个浮动罗盘，然后用摇杆(鼠标)来控制选择的物品,放开触发键后就触发,,就类似那种感觉!是不是很棒!
![image](https://user-images.githubusercontent.com/57837939/144698401-b2bebd2e-3c2a-442d-b093-b7a5d761e4c9.png)
![image](https://user-images.githubusercontent.com/57837939/144730609-694cd826-0214-4f74-85ab-7213158ff688.png)
既能手势直接触发 也能罗盘触发(圆形限定根据鼠标移动高亮选定框选中后停止移动鼠标松开罗盘触发键即可触发手势),
当然,这个属于展望,不是刚需,但做出来肯定是如虎添翼了!

**Comment by dylumian** [comment]
你这个畅想让我想到了maya的甩线

**Comment by superDMS** [comment]
> 你这个畅想让我想到了maya的甩线

是的  手势脑图是让越来越多的手势分支可视化  更易管理  巩固记忆  一目了然    对构建一套数量较多的手势也有正面作用! 现版本的手势管理对越来越多的手势来讲似乎少了点什么   只能靠搜索栏  整体调整的条理性欠缺  有时候甚至忘了手势也忘了名称  得拉下去慢慢找

**Comment by superDMS** [comment]
2.8的更新,,,,, 自定义菜单果然来了!  很棒! 能否加入一个导入现有动作的入口?勾选导入现有手势的动作? 毕竟重新添加n多手势确实不太方便,且有时候小小的修改一个动作的双向修改 ,
我自己想的两个逻辑规则:
1.在自定义菜单中增加导入现有动作手势按钮,,导入后自动继承手势名+动作(同步共用代码,修改手势那边的动作则导入到菜单里的动作也随之改变,,,或者导入后无关联也可!甚至可自定义是否与导入前的手势动作同步!),,,菜单则根据设定的列数自动填充排列!  
2.  或者在手势选项中加入一个推送按钮,可把单独手势动作可推送到指定的已创建的自定义菜单里. 不用再次到菜单中新建一个相同重复的手势.   
这样就真正做到了即可手势触发,也可菜单呼出触发!   现有手势动作也有更好复用性,,,!!

**Comment by yingDev** [comment]
这可能得3.0去了...

**Comment by superDMS** [comment]
> 这可能得3.0去了...

嘿嘿  没问题  !未来可期! !加油,老大

**Comment by eliaszon** [comment]
朋友你真的应该试试Quicker这个软件，里面的动作轮盘正是你所需要的。而且还自带了鼠标手势。

**Comment by superDMS** [comment]
> 朋友你真的应该试试Quicker这个软件，里面的动作轮盘正是你所需要的。而且还自带了鼠标手势。

用了一下quicker, 发现能移植过来的功能确实不少,, 比如轮盘菜单, 文本指令   都挺好用, 手势方面wg2秒杀 quicker,况且quicker收费限制太多了!  希望老大逐步把轮盘菜单, 文本指令 这两样超实用的东东加入wg2,这些功能会发展壮大咱的wg2!!

**Comment by superDMS** [comment]
https://getquicker.net/kc/manual/Doc/textcommand    轮盘可能难做一些,如果要做先做文字指令吧! 这个真心好用而且应该比较好做,相当于是先录入命令再按触发, 颠倒过来, 有些长文本用这个非常舒服!
